easybuild/easyconfigs/m/miniprot/miniprot-0.13-GCCcore-12.3.0.eb

```diff
@@ -15,8 +15,8 @@ source_urls = ['https://github.com/lh3/miniprot/archive']
 sources = [{'download_filename': 'v%(version)s.tar.gz', 'filename': SOURCE_TAR_GZ}]
```

Comment:
Using GCCcore those need to be specified if they are actually needed (it seems the `13.2.0` one worked also without binutils)

```
crivella@crivella-desktop:~$ module load GCCcore/13.2.0 
crivella@crivella-desktop:~$ module list

Currently Loaded Modules:
  1) easybuild_env/custom (S)   2) GCCcore/13.2.0
```

Also i would leave fixing the file from the `12.3.0` tc to a separate PR

Reply:
Okay... then I do not understand the tests complaining before the commit